projects/packages/connection/composer.json

```diff
@@ -17,7 +17,9 @@
 		"automattic/wordbless": "@dev",
 		"yoast/phpunit-polyfills": "1.1.0",
 		"brain/monkey": "2.6.1",
-		"automattic/jetpack-changelogger": "@dev"
+		"automattic/jetpack-changelogger": "@dev",
+		"automattic/jetpack-licensing": "@dev",
+		"automattic/jetpack-sync": "@dev"
```

Comment:
Is it ok that the Connection package requires sync, while the Sync package requires the connection package?

Reply:
It should be ok.

* Composer doesn't care as long as it can come up with a consistent set of packages, which it can.
* Our build scripts are ok with it now as long as at least one of the deps is marked as "test-only", thanks to #37140.
* Is there anything else you're concerned about?